### PR TITLE
r2r: Remove diff header and colorize @@ line

### PR DESCRIPTION
**Pull request by kazarmy**
**Your checklist for this pull request**
- [X] I've read the [guidelines for contributing](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md) to this repository
- [X] I made sure to follow the project's [coding style](https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md#code-style)
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [radare2 book](https://github.com/radareorg/radare2book) with the relevant information (if needed)

**Detailed description**

This pr does the following:

Windows before (`git diff`):

Windows_@@_before

Windows after:

Windows_@@_after

----

Linux before (`diff`):

Linux_@@_before

Linux after:

Linux_@@_after

----

In other words, this pr removes the diff header (since there's no information there) and thus shortens the output, and also colorizes the `@@` line to differentiate it.

**Test plan**

Eyeball the difference?

**Closing issues**

...

### CODE REVIEW

**Comment by XVilka** [comment]
Spurious fail on OpenBSD again:
```
[XX] db/cmd/cmd_meta vars commenting
R2_NOPLUGINS=1 radare2 -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'e asm.arch=x86
e asm.bits=64

s 0x10
af

afvb 8 arg_8h int
afvb -8 var_8h uint32_t
afvs 8 var_sp_8h uint32_t
afvr rax var_rax uint64_t

Cvb 0x8 comment for arg_8h
Cvb -8 comment for var_8h
Cvr var_rax comment for var_rax
Cvs var_sp_8h comment for var_sp_8h

pd1~comment
?e --
Cvb
Cvb*
?e --
Cvr
Cvr*
?e --
Cvs
Cvs*
?e --

Cvb- arg_8h
pd1~comment
?e --
Cvb- -8
pd1~comment
?e --
Cvs- 8
pd1~comment
?e --
Cvr- var_rax
pd1~comment
?e --

' malloc://1024
-- stdout
@@ -8,19 +8,3 @@
"Cvb arg_8h base64:Y29tbWVudCBmb3IgYXJnXzho @ 0x00000010"
"Cvb var_8h base64:Y29tbWVudCBmb3IgdmFyXzho @ 0x00000010"
--
-var_rax : comment for var_rax
-"Cvr var_rax base64:Y29tbWVudCBmb3IgdmFyX3JheA== @ 0x00000010"
---
-var_sp_8h : comment for var_sp_8h
-"Cvs var_sp_8h base64:Y29tbWVudCBmb3IgdmFyX3NwXzho @ 0x00000010"
---
-|           ; var uint32_t var_8h @ rbp-0x8    ; comment for var_8h
-|           ; arg uint32_t var_sp_8h @ rsp+0x8    ; comment for var_sp_8h
-|           ; arg uint64_t var_rax @ rax    ; comment for var_rax
---
-|           ; arg uint32_t var_sp_8h @ rsp+0x8    ; comment for var_sp_8h
-|           ; arg uint64_t var_rax @ rax    ; comment for var_rax
---
-|           ; arg uint64_t var_rax @ rax    ; comment for var_rax
---
---


-- exit status: -1
```
